.travis.yml

```diff
@@ -7,10 +7,6 @@ rvm:
   - '2.6'
   - '2.5'
   - '2.4'
-  - '2.3'
```

Comment:
Please don't drop these while the gemspec says we still support 2.0.

Reply:
updated it